<?xml version="1.0" ?>
<sdf version="1.6">
  <world name="default">

    <include>
      <uri>model://ground_plane</uri>
    </include>

    <include>
      <uri>model://sun</uri>
    </include>

    <!-- Link attacher plugin for pick and place -->
    <plugin name="gazebo_ros_link_attacher" filename="libgazebo_ros_link_attacher.so"/>

  </world>
</sdf>
